config/routes.rb

```diff
@@ -9,5 +9,5 @@
     end
   end 
   get '/recipes', to: 'recipes#index'
```

Comment:
You created a new route for public_recipes so please remove the other one because it now becomes recipes for a specific user

Reply:
@ahmedadel56, I believe we still need this because it's a requirement in the wireframe.